src/perfume.ts

```diff
@@ -198,7 +206,7 @@ export default class Perfume {
     // Get duration and change it to a two decimal value
     const duration = this.perf.measure(metricName, metric);
     const duration2Decimal = parseFloat(duration.toFixed(2));
-    this.metrics.delete(metricName);
+    delete this.metrics.metricName;
```

Comment:
This one should probably be `delete this.metrics[metricName]`

Reply:
both is valid syntax. see https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Operators/delete

Reply:
This example should help explain my point
<img width="338" alt="Screen Shot 2019-09-10 at 9 42 24 PM" src="https://user-images.githubusercontent.com/829214/64668814-eac8b500-d413-11e9-9c41-d4c81e66f17e.png">

Also, I should probably add a test to prevent this issue.

Reply:
Oh, I see your point. Will fix that...

Reply:
have fixed it. added a test to check proper deletion of metrics. this test fails with the previous implementation and passes with the fix as expected. :)